packages/core/workers/test/workerfarm.js

```diff
@@ -2,6 +2,8 @@ const assert = require('assert');
 const WorkerFarm = require('../index');
 
 describe('WorkerFarm', () => {
+  this.timeout(10000);
```

Comment:
Probably better to change this for all workerfarm tests by changing `packages/core/workers/test/mocha.opts`

Reply:
True - I’m still learning how the code is organized.
Thanks for the feedback.